### PR TITLE
Rg letters palindrome

### PR DESCRIPTION
**Pull request by RodoGalofre**
here you can calculate if a word entered by the user is a palindrome

### CODE REVIEW

File: LettersPalindrome.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Metodo de entrada 
+def input_data(message)
+    puts "Ingrese la palabra"
+    palabra = gets.chomp
+    message = "¿quieres seguir?"
+    calcula_palin(palabra) 
+end
+
+# metodo que calcula si la palabra es un palindromo
+def calcula_palin(palabra)
```
Comment (march23) [inline LettersPalindrome.rb]: Let's make this more readeable, yes `reverse` does the job , you could make this work in one single line:
```
def palindrome?
  word.reverse == 'word' ? true : false
end
```
then you return a boolean to validate it

File: LettersPalindrome.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+def calcula_palin(palabra)  
+  rever = palabra.reverse
+  puts (palabra == rever) ? "la palabra #{palabra.upcase} es un palindromo" :  "#{palabra.upcase} no es palindromo "
+  
+  validar_seguir()
+end  
+
+
+# Metodo para confirmar si el usuario desea introducir más palabras
+def validar_seguir()
```
Comment (march23) [inline LettersPalindrome.rb]: avoid  mixing `english` with `spanish` on method, vars and code definitions

**Review by march23 — COMMENTED** [review]
- Avoid mixing languages on method names, vars, commits , is not a good practice. Review conventions to name branches and commits too
- a method name should be  called after what it does
- review how to print outputs without using puts 
- add description to you're PR's

**Comment by danielosorio83** [comment]
@RodoGalofre borraste el archivo?

**Review by danielosorio83 — APPROVED** [review]

**Pull request merged**